apm-sniffer/apm-agent-core/src/main/java/org/apache/skywalking/apm/agent/core/conf/Constants.java

```diff
@@ -26,4 +26,8 @@ public class Constants {
     public static String EMPTY_STRING = "";
 
     public static char SERVICE_NAME_PART_CONNECTOR = '|';
+
+    // The name of the layer that represents agent-installed services,
+    // which is defined at https://github.com/apache/skywalking/blob/master/oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/analysis/Layer.java
```

Comment:
If  you want to put link in comments, please use perm link.

Reply:
ok